auto-configurations/models/chat/memory/repository/spring-ai-autoconfigure-model-chat-memory-repository-jdbc/src/main/java/org/springframework/ai/model/chat/memory/repository/jdbc/autoconfigure/JdbcChatMemoryRepositoryProperties.java

```diff
@@ -17,17 +17,21 @@
 package org.springframework.ai.model.chat.memory.repository.jdbc.autoconfigure;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.sql.init.DatabaseInitializationMode;
```

Comment:
We have decided that the core of Spring AI will not depend on spring boot, for the chat repository work, we created our own enum for this.

Reply:
oops, my bad, I mean thtat for the non autoconfig code.